xiaomi_gateway/__init__.py

```diff
@@ -46,10 +46,10 @@ def discover_gateways(self):
             host = gateway.get('host')
             port = gateway.get('port')
             sid = gateway.get('sid')
+            discovery_retries = gateway.get('discovery_retries', 1)
```

Comment:
Should we use a better default here? 4 retries f.e.

Reply:
3 minimum in my opinion.
It shouldn't make initialization much longer anyway?

Reply:
That would make much more sense, however I wanted it to work as before by default.

Reply:
The retry loop breaks early if the zigbee device is discovered properly. The startup/discovery will take some more time for the bad case only.